test/test.js

```diff
@@ -3,64 +3,76 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inNode = require('detect-node');
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = inNode ? require('jsdom') : null;
```

Comment:
Again, I think we could detect for `inNode` here by doing something like `typeof document === 'undefined' ? require('jsdom') : document`